scripts/helpers.sh

```diff
@@ -62,3 +62,49 @@ command_exists() {
   local command="$1"
   command -v "$command" &> /dev/null
 }
+
+get_tmp_dir() {
+  local tmpdir="${TMPDIR:-${TMP:-${TEMP:-/tmp}}}"
+  if [ ! -d "$tmpdir" ]; then
+    local tmpdir=~/tmp
+    mkdir -p $tmpdir
+  fi
```

Comment:
`mkdir -p` is already run elsewhere

```suggestion
  [ -d "$tmpdir" ] || local tmpdir=~/tmp
```